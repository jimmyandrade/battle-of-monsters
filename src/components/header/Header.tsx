import { FC } from 'react'
import { Container } from '../container'
import { HeaderRoot } from './Header.styles'

export const Header: FC = () => (
  <HeaderRoot>
    <Container>
      <h1>Battle of Monsters</h1>
    </Container>
  </HeaderRoot>
)
