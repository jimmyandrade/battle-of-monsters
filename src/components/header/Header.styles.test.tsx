import { HeaderRoot } from './Header.styles'
import { render } from '@testing-library/react'

describe('Header styles', () => {
  describe('Root component', () => {
    it('should match snapshot', () => {
      const { container } = render(<HeaderRoot />)
      expect(container).toMatchSnapshot()
    })
  })
})
