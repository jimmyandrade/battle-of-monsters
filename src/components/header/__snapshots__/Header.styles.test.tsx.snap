// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header styles Root component should match snapshot 1`] = `
.c0 {
  padding-block-end: 35px;
  padding-block-start: 69px;
}

<div>
  <header
    class="c0"
  />
</div>
`;
